src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -131,10 +131,6 @@ public static Properties createProperties(Properties inputProp) {
               Integer.parseInt(properties.get(Constants.PORT).toString())));
     }
 
-    if (!properties.containsKey(Constants.ROLE)) {
-      throw new SFException(ErrorCode.MISSING_CONFIG, "role");
```

Comment:
should we throw a logInfo to indicate that the default role is being used?

Reply:
Good to have one, added [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/0b2c10b0466c2d0f01424faedbe3f74f4e0993cb/src/main/java/net/snowflake/ingest/utils/Utils.java#L134).

Reply:
Fixed.